contracts/razorPool.sol

```diff
@@ -36,9 +36,9 @@ contract LPTokenWrapper {
     }
 }
 
-contract Unipool is LPTokenWrapper, IRewardDistributionRecipient {
+contract razorPool is LPTokenWrapper, IRewardDistributionRecipient {
```

Comment:
no need to change name

Reply:
ok

Reply:
@hrishikeshio I have made changes. Can we merge this ?